mkdocs.yml

```diff
@@ -172,7 +172,6 @@ nav:
     - Omnissa Intelligence SDK Constants: ./Apple/ws1constants.md
     - SDK Modules:
       - Logging Crashes: ./Apple/ios-crash.md
-      - CrashData: ./Apple/ios-crash.md
```

Comment:
MkDocs rendered this as

    SDK Modules
        Logging Crashes
        Logging Crashes